lib/renderJs.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+'use strict';
+
+const SassValue = require('./SassValue');
+
+function createJsonMap(data) {
+  return JSON.stringify(
+    data,
+    (key, value) =>
+      value instanceof SassValue
+        ? value.fallback
+          ? value.fallback
+          : undefined
```

Comment:
Accessing a nonexistent property in JS returns `undefined` already - so it seems like this is superfluous, unless you want to collapse falsey values (`''`, `0`, `false`, `null`) to `undefined` as well. Even then, I argue it'd be easier to read as `value.fallback || undefined` since it omits a ternary.

Reply:
Yeah, good call. Will fix.